src/Ai/TestCommunicator.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace App\Ai;
+
+use App\Suggestion\BookPromptInterface;
+use App\Suggestion\SummaryPrompt;
+use App\Suggestion\TagPrompt;
+use Symfony\Component\DependencyInjection\Attribute\AutoconfigureTag;
+use Symfony\Component\DependencyInjection\Attribute\Autowire;
+
+#[AutoconfigureTag('app.ai_communicator', ['priority' => 0])]
+class TestCommunicator implements AiCommunicatorInterface
+{
+    public function __construct(
+        #[Autowire(param: 'kernel.environment')]
+        private readonly ?string $environment,
+    ) {
+    }
+
+    public function isEnabled(): bool
+    {
+        return $this->environment === 'test';
```

Comment:
Do you want to inject this service only in test env ? Because you could use a `when@test`  in the services.yaml.